scripts/setup_haskell.hs

```diff
@@ -164,18 +170,14 @@ setupHaskell = do
                       (Turtle.begins ("#" <|> "user-message" <|> "  ") *>
                        pure "")
                       (Turtle.input "stack.yaml.bak")))
-          versionedHelperDeps <-
-            fmap Turtle.lineToText <$>
-            Turtle.fold
-              (mfilter
-                (filterHelperDeps . Turtle.lineToText)
-                (Turtle.inshell "stack list-dependencies --separator -"
```

Comment:
I don't think we want to remove the `stack list-dependencies` invocation here.  That was done so that we'd get the specific package version strings, i.e. `acme-some-package-0.1.0.7` instead of just `acme-some-package`.  `helperDependencies` just lists the package name without version info.

Reply:
We don't need them to be versioned, stack installs the appropriate version based on the config/resolver.

Reply:
Oh right, makes total sense.  👍